server/routes.js

```diff
@@ -112,12 +112,29 @@ module.exports = [{
     }
   }
 }, {
-  // static routes using dist/, yay grunt
   method: 'GET',
-  path: '/assets/{param*}',
+  path: '/images/{param*}',
   handler: {
     directory: {
-      path: path.join(__dirname, '..', config.get('server.staticPath')),
+      path: path.join(__dirname, '..', config.get('server.staticPath'), 'images'),
+      listing: config.get('server.staticDirListing')
+    }
+  }
+}, {
+  method: 'GET',
+  path: '/scripts/{param*}',
+  handler: {
+    directory: {
+      path: path.join(__dirname, '..', config.get('server.staticPath'), 'scripts'),
+      listing: config.get('server.staticDirListing')
+    }
+  }
+}, {
+  method: 'GET',
+  path: '/styles/{param*}',
+  handler: {
+    directory: {
+      path: path.join(__dirname, '..', config.get('server.staticPath'), 'styles'),
```

Comment:
@nchapman @6a68 -- Is this what you kids were thinking about re: routing? I basically created a whitelist of routes for /images/, /scripts/, and /styles/. Or we could try and create a "*" handler which captures all unmatched routes and points them to `config.get('server.staticPath')` and make sure that 404s still route properly and don't get swallowed by some magic route.